test/selenium/pages/ContributionsLanding.scala

```diff
@@ -14,6 +14,8 @@ case class ContributionsLanding(region: String)(implicit val webDriver: WebDrive
 
   private val oneOffButton = id("qa-one-off-toggle")
 
+  private val otherAmountField = className("component-number-input__input")
```

Comment:
Is it worth adding a QA id to the element as we've done to the ones above? When relying on a class I think there's a reasonable chance that someone might change it during a refactor, without realising that the tests are relying on it. With a specific QA-related id, anyone touching the code will know they should avoid changing it/preserve it through any changes.